pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -193,9 +193,11 @@ func withNetworkIfacesResources(vmi *v1.VirtualMachineInstance, domainSpec *api.
 		return nil, err
 	}
 	domainSpecWithoutIfacePlaceholders.Devices.Interfaces = domainSpec.Devices.Interfaces
-	domainSpecWithoutIfacePlaceholders.DeepCopyInto(domainSpec)
+	// Only the devices are taken into account because some parameters are not assured to be returned when
+	// getting the domain spec (e.g. the `qemu:commandline` section).
```

Comment:
Do you know why the `qemu:commandline` is missing? Is it a libvirt bug?

Reply:
I think it is a bug, I will discuss it with @andreabolognani  offline.